libs/core/mpi_base/src/mpi_environment.cpp

```diff
@@ -33,16 +34,23 @@ namespace hpx { namespace util {
             std::string mpi_environment_strings =
                 cfg.get_entry("hpx.parcel.mpi.env", default_env);
 
-            typedef boost::tokenizer<boost::char_separator<char>> tokenizer;
             boost::char_separator<char> sep(";,: ");
-            tokenizer tokens(mpi_environment_strings, sep);
-            for (tokenizer::iterator it = tokens.begin(); it != tokens.end();
-                 ++it)
+            boost::tokenizer<boost::char_separator<char>> tokens(
+                mpi_environment_strings, sep);
+            for (auto const& tok : tokens)
             {
-                char* env = std::getenv(it->c_str());
+                char* env = std::getenv(tok.c_str());
                 if (env)
+                {
+                    LBT_(debug)
+                        << "Found MPI environment variable: " << tok << "="
+                        << std::string(env) << ", enabling MPI support\n";
                     return true;
+                }
             }
+
+            LBT_(info) << "No know MPI environment variable found, disabling "
```

Comment:
```suggestion
            LBT_(info) << "No known MPI environment variable found, disabling "
```
sorry... looks good otherwise. Thank you!